task/06-conditions-n-loops-tasks.js

```diff
@@ -81,8 +98,12 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a, b, c) {
+    if (a + b > c && a + c > b && b + c > a) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
we can simply return (a + b > c && a + c > b && b + c > a)
Please, see https://javascript.info/logical-operators